playbooks/templates/rax-maas/ceph_cluster_stats.yaml.j2

```diff
@@ -2,8 +2,11 @@
 {% set label = "ceph_cluster_stats" %}
 {% set check_name = label+'--'+inventory_hostname %}
 {% set ceph_args = [maas_plugin_dir + "/ceph_monitoring.py", "--name", "client.raxmon", "--keyring", "/etc/ceph/ceph.client.raxmon.keyring"] %}
-{% if maas_rpc_legacy_ceph | bool %}
-{%   set _ = ceph_args.extend(["--container-name", container_name]) %}
+{% if (maas_rpc_legacy_ceph | bool) or deploy_osp %}
```

Comment:
Do we need `deploy_osp|default(False)` here?

Reply:
Good catch,  done.